### PR TITLE
refactor: upgrade trip-details, use its translation strings

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Makes use of https://github.com/opentripplanner/otp-ui/pull/569. Uses translation strings from otp ui instead of duplicating them.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/narrative/default/default-itinerary.js [hunk 1]
```diff
@@ -123,9 +123,7 @@ const ITINERARY_ATTRIBUTES = [
       const fareCurrency = itinerary.fare?.fare?.regular?.currency?.currencyCode
       const fare = fareInCents === null ? null : fareInCents / 100
```
Comment (binh-dam-ibigroup) [inline lib/components/narrative/default/default-itinerary.js:124]: I suppose this logic could be reused in the trip-details component to handle cases where cents show as `-1`. (If that's the case, trip details and the itinerary summary render different things).

**Review by binh-dam-ibigroup — APPROVED** [review]
One small change.

File: lib/components/narrative/default/default-itinerary.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/narrative/default/default-itinerary.js:124]: Noted in https://github.com/opentripplanner/otp-ui/issues/576.

Comment (miles-grant-ibigroup) [reply lib/components/narrative/default/default-itinerary.js:124]: Good call thanks. Yes I think this should be in core-utils eventually

**Review by AdrianaCeric — APPROVED** [review]
This is a great change, good catch

**Pull request merged**